### DISCUSSION TITLE
UI Extensions stop working after Neovim 0.6

### DISCUSSION THREAD

**Discussion by baobaoit** [mfussenegger/nvim-jdtls#199] [Q&A] [answered]
Hi @mfussenegger,

The `UI Extensions` is stopped working after I was up-to-date. I see this on your wiki page:

> If you are on Neovim >= 0.6, nvim-jdtls uses `vim.ui.select`, that means plugins providing a `vim.ui.select` implementation will automatically be utilized.

I used `telescope.nvim` before, so how can I migrate this with Neovim 0.6?

I tried this:
```lua
require("jdtls.ui").pick_one_async = vim.ui.select
```

But it shows me:
![image](https://user-images.githubusercontent.com/31947091/145752240-2d407d17-f112-4652-a2d2-ac17107f1bf6.png)

Would you please give me an advice?

Thank you.

**Comment by mfussenegger** [comment, marked as answer]
Looks like you could use https://github.com/nvim-telescope/telescope-ui-select.nvim 
The built-in client and nvim-jdtls use `vim.ui.select` and if telescope provides the implementation for it, you should get telescope prompts for the selection
